package.json

```diff
@@ -50,6 +56,7 @@
     "terser": "5.16.1",
     "typescript": "4.9.4",
     "vite": "4.0.4",
+    "vue": "^3.2.37",
```

Comment:
We shouldnt need vue in the dev deps if its also in the regular deps, afaik

Reply:
done